qcodes/instrument_drivers/AlazarTech/ats_api.py

```diff
@@ -21,17 +21,19 @@
 U8 = ctypes.c_uint8
 U16 = ctypes.c_uint16
 U32 = ctypes.c_uint32
+C_LONG = ctypes.c_long
```

Comment:
nice! `AlazarBeforeAsyncRead` also uses c_long so shall it now use this alias as well?